examples/llm/src/service.rs

```diff
@@ -73,23 +76,24 @@ impl Service for LlmService {
     type Parameters = ();
 
     async fn new(runtime: ServiceRuntime<Self>) -> Self {
-        LlmService { runtime }
+        let raw_weights = runtime
+            .fetch_url("https://huggingface.co/karpathy/tinyllamas/resolve/main/stories42M.bin");
+        info!("got weights: {}B", raw_weights.len());
```

Comment:
Perhaps improve this message a little? E.g. `"Model was downloaded: {} bytes"`

Reply:
I'll improve logging in a separate PR :+1: